#!/usr/bin/env python3
PACKAGE = "docking_server"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("kp_linear", double_t, 0, "A double parameter",    0.4, 0,   10)
gen.add("kd_linear", double_t, 0, "A double parameter",    0.05, 0,   10)
gen.add("kp_angular", double_t, 0, "A double parameter",   0.4, 0,   10)
gen.add("kd_angular", double_t, 0, "A double parameter",   0.05, 0,   10)


exit(gen.generate(PACKAGE, "docking_server", "Config"))